spark-connector/spark-common/src/main/java/org/apache/gravitino/spark/connector/iceberg/IcebergPropertiesConstants.java

```diff
@@ -58,6 +59,8 @@ public class IcebergPropertiesConstants {
 
   static final String GRAVITINO_ICEBERG_CATALOG_BACKEND_REST = "rest";
 
+  public static final String ICEBERG_CATALOG_BACKEND_CUSTOM = "custom";
```

Comment:
We have already defined `custom` catalog backend in `IcebergCatalogBackend`, could you do a refactor to moving `IcebergCatalogBackend` to `catalogs/catalog-common` module, so the spark connector could reuse the class.